eng/pipelines/docindex.yml

```diff
@@ -33,7 +33,8 @@ jobs:
 
   - job: UpdateDocsMsBuildConfig
     pool:
-      name: $(LINUXPOOL)
+      name: azsdk-pool
```

Comment:
Why not use the variables? It might also be interesting to add the demand to the image.yml variables.

Reply:
Tested and the demand can be placed in a variable and it can reference variables (when the values are defined before job run time).